fleetspeak/src/admin/cli/cli.go

```diff
@@ -184,7 +184,7 @@ type byTimestamp []*spb.ClientContact
 
 func (b byTimestamp) Len() int           { return len(b) }
 func (b byTimestamp) Swap(i, j int)      { b[i], b[j] = b[j], b[i] }
-func (b byTimestamp) Less(i, j int) bool { return timestamp(b[i]).Before(timestamp(b[j])) }
+func (b byTimestamp) Less(i, j int) bool { return timestamp(b[i]).After(timestamp(b[j])) }
```

Comment:
This is obvious once you see it, but still deserves a comment, i.e. // Note the reversed chronological sort order

Reply:
Updated struct comment.